core/src/main/java/org/apache/calcite/rel/rules/CoreRules.java

```diff
@@ -847,6 +847,10 @@ private CoreRules() {}
   public static final AggregateValuesRule AGGREGATE_VALUES =
       AggregateValuesRule.Config.DEFAULT.toRule();
 
+  /** Rule that applies an {@link Aggregate} to a distinct {@link Values}. */
```

Comment:
/** Rule that applies {@link Aggregate} and {@link Values} to a distinct {@link Values}. */ Is that right?

Reply:
Yes，your suggestion is more suitable.  had changed as it. @xiedeyantu